0-bootstrap/README-Jenkins.md

```diff
@@ -142,69 +154,49 @@ You arrived to these instructions because you are using the `jenkins_bootstrap`
 
 1. Run terraform commands.
     - After the credentials are configured, we will create the `cft-seed` project (which contains the GCS state bucket and Terraform custom service account) and the `prj-cicd` project (which contains the Jenkins Agent, its custom service account and where we will add VPN configuration)
-    - **Use Terraform 0.12.24** to run the terraform script with the commands below:
     - **WARNING: Make sure you have commented-out the `cloudbuild_bootstrap` module and enabled the `jenkins_bootstrap` module in the `./main.tf` file**
+    - **Use Terraform 0.12.24** to run the terraform script with the commands below
     ```
     terraform init
     terraform plan
     terraform apply
     ```
-    - The Terraform script will take about 10 to 15 minutes. Once it finishes, note that communication between on-prem and the `prj-cicd` project won’t happen yet - you will configure the VPN network connectivity in step [III. Create VPN connection](#III.-Create-VPN-connection).
+    - The Terraform script will take about 10 to 15 minutes. Once it finishes, note that communication between on-prem and the `prj-cicd` project won’t happen yet - you will configure the VPN network connectivity in step [III. Create VPN connection](#III-Create-VPN-connection).
 
 1. Move Terraform State to the GCS bucket created in the seed project
-    - Run this command to copy the `backend.tf` file and update the GCS bucket name. Replace the `TF_STATE_GCS_BUCKET_NAME` with the name of your bucket (you can run `terraform output` to find these values).
-    ```
-    TF_STATE_GCS_BUCKET_NAME=`(terraform output gcs_bucket_tfstate)`
-    mv backend.tf.example backend.tf
-    sed -i "s/UPDATE_ME/$TF_STATE_GCS_BUCKET_NAME/" backend.tf
-    ```
-
-   **If using MacOS:**
-    ```
-    TF_STATE_GCS_BUCKET_NAME=`(terraform output gcs_bucket_tfstate)`
-    mv backend.tf.example backend.tf
-    sed -i ".bak" "s/UPDATE_ME/$TF_STATE_GCS_BUCKET_NAME/" backend.tf
-    rm backend.tf.bak
-    ```
+   1. Rename `backend.tf.example` to `backend.tf`
+   1. Run `terraform output gcs_bucket_tfstate` to get the tfstate bucket name
```

Comment:
Should run this command *before* the rename to avoid errors about consistency.

Reply:
Done.